serverless.yml

```diff
@@ -66,15 +67,21 @@ functions:
       - http: 'GET /art'
         cors: true
 
+  searchArt:
+    handler: index.handler
+    events:
+      - http: 'POST /search/art'
+        cors: true
+
   createArt:
     handler: index.handler
     events:
       - http: 'POST /art'
         cors: true
-    authorizer:
-        type: COGNITO_USER_POOLS
-        authorizerId:
-            Ref: ApiGatewayAuthorizer
+    #authorizer:
```

Comment:
why is this commented out again?

Reply:
We make it public and once it work we add the permission